config/printer-creality-ender5-s1-2023.cfg

```diff
@@ -0,0 +1,165 @@
+# Creality Ender 5 S1
+#
+# printer_size: 220x220x280
+# To use this config, during "make menuconfig" select the STM32F401
+# with a "64KiB bootloader" and serial (on USART1 PA10/PA9)
+# communication.
+#
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
```

Comment:
Sometimes these boards needs the firmware putting in a specific directory on the SD card, usually STM32F4_UPDATE can you confirm if this is the case?

Thanks
James

Reply:
You are correct.  I will update the header comments, and include the fact that in addition to the `STM32F4_UPDATE` directory, the filename must differ from whatever filename the current firmware used.